app/templates/src/app/main/__angular-material.html

```diff
@@ -1,8 +1,9 @@
 <div layout="vertical" layout-fill>
-
-  <header ng-include="'components/navbar/navbar.html'"></header>
-
   <md-content>
+    <md-toolbar>
+      <header ng-include="'components/navbar/navbar.html'"></header>
```

Comment:
Yup... I didn't notice the md-toolbar in the navbar.html.  I think the main thing is that the md-toolbar just needed to be in the md-content